README.md

```diff
@@ -164,6 +164,47 @@ This is then the built-in version from the brower.
 Typical effect is that it is shown slightly delayed and that is looks a bit primitive.
 The only other allowed value is "none", with the effect that no tooltips are shown at all.
 
+## Separate SaltGUI host
+In some specific environments you might not be able to server SaltGUI directly from salt-api.
+In this case you might want to configure web server (for example NGINX) to server SaltGui 
```

Comment:
this->that
to configure web server -> to configure a web server
to server -> to serve